app/controllers/events_controller.rb

```diff
@@ -0,0 +1,34 @@
+class EventsController < ApplicationController
+  def index
+    @upcoming_events = Event.upcoming
+    @prev_events = Event.past
+  end
+
+  def show
+    @event = Event.where(creator_id: current_user.id)
+  end
+
+  def new
+    @event = current_user.created_events.build
+  end
+
+  def create
+    @event = current_user.created_events.build(event_params)
+    @event.creator_id = current_user.id
+
+    if @event.save
+      create_invitation(params[:user_ids], @event.id)
+      flash.now[:notice] = 'Event was successfully created.'
```

Comment:
This should be `flash[:notice]`. Check [here](http://spaghettirefactory.blogspot.com/2015/03/rails-flash-vs-flashnow.html#:~:text=Boiled%20down%20to%20the%20very,page%20after%20clicking%20a%20link.) for more clarification.